src/main/groovy/org/shipkit/internal/gradle/notes/ReleaseNotesPlugin.java

```diff
@@ -106,6 +104,6 @@ private static void configureDetailedNotes(final UpdateReleaseNotesTask task,
         task.setPreviousVersion(project.getExtensions().getByType(VersionInfo.class).getPreviousVersion());
 
         task.setReleaseNotesData(releaseNotesFetcher.getOutputFile());
-        task.setContributorsDataFile(contributorsFetcher.getOutputFile());
+        task.setContributorsDataFile(contributorsFetcher.getOutputs().getFiles().getSingleFile());
```

Comment:
Is there any particular reason you use "outputs.files.singleFile" instead of "outputFile"? I think the latter is more conventional and simpler.